jest.config.js

```diff
@@ -87,18 +89,18 @@ module.exports = {
   // An enum that specifies notification mode. Requires { notify: true }
   // notifyMode: "failure-change",
 
-  // A preset that is used as a base for Jest's configuration
+  // A preset that is used as a base for Jest"s configuration
```

Comment:
Yes, the reason is that this application is linted with [standard](https://standardjs.com/).

But since our Scrivito projects use different lint rules, I want to switch to these rules here too. But later.

Reply:
i would just fix for sake of travis, cuz who knows when 'later' would happen...